plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/wizard/AltConfigWizard.java

```diff
@@ -238,12 +238,6 @@ public boolean performFinish() {
 		IResource targetFile = (IResource) currentPage.getSelectedResource().getFirstElement();
 
 		String taskName = selectedTask.getName();
-		JOptionPane optionPane = new JOptionPane("CogniCrypt is now generating code that implements " + selectedTask.getDescription() + "\ninto file " + ((targetFile != null)

```

Comment:
How much does this affect runtime?

Reply:
The actually issue was the dialog appears in the background and doesn't close when the code generation process is completed. Have added it back and have set `visibility` to false after generation is finished.